### PR TITLE
Adding jieba3k

### PR DESCRIPTION
**Pull request by ganeshhubale**
This is my patch for jieba3k. Please review.

### CODE REVIEW

File: recipes/jieba3k/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: 'python setup.py install '
```
Comment (synapticarbors) [inline recipes/jieba3k/meta.yaml]: It looks like this package is python 3 only so add:

```
build:
   ...
   skip: True  # [py2k]
```

File: recipes/jieba3k/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
```
Comment (synapticarbors) [inline recipes/jieba3k/meta.yaml]: consider adding `noarch: python`:

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages

File: recipes/jieba3k/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: Chinese Words Segementation Utilities
+  description: " "Jieba" (Chinese for "to stutter") Chinese text segmentation: built to be the best Python Chinese word segmentation module."
+  dev_url: 'http://github.com/fxsjy'
+
+extra:
+  recipe-maintainers: 
+  - ganeshhubale
```
Comment (synapticarbors) [inline recipes/jieba3k/meta.yaml]: I think you need to indent this, which may be the source of the parsing error of the `meta.yaml`

Comment (ganeshhubale) [reply recipes/jieba3k/meta.yaml]: I updated meta.yaml as per your suggestions. Please review.
Thank you,

**Comment by ganeshhubale** [comment]
I am not getting actual problem. Please suggest changes.

**Comment by synapticarbors** [comment]
Fixed license file issue with the recipe. Raised an issue upstream about including it in the source distribution:

https://github.com/fxsjy/jieba/issues/545

**Pull request merged**